### DISCUSSION TITLE
Wizard Pics Only!

### DISCUSSION THREAD

**Discussion by ardisrtilson** [nss-day-cohort-47/C47-Discussions#26] [Topix.random]
![st,small,507x507-pad,600x600,f8f8f8 u1](https://user-images.githubusercontent.com/67875837/110167956-1f74bd00-7dbc-11eb-8af3-59ed2a40bb97.jpg)

**Comment by BryanNilsen** [comment]
![image](https://user-images.githubusercontent.com/43187473/110168803-50a1bd00-7dbd-11eb-88e7-774384c9acd0.png)

**Comment by AutumnFoto** [comment]
![harry](https://user-images.githubusercontent.com/67810121/110176842-906ea180-7dc9-11eb-8e8a-00327034e734.jpg)

**Reply by BrysonMG** [reply]
"You're a wizard Harry"

"You're a hairy wizard"

**Comment by jrossgage** [comment]
![FInalWizard](https://user-images.githubusercontent.com/78052825/110177913-52727d00-7dcb-11eb-8aa8-2f4670d06bbc.jpg)
